evalml/automl/automl_search.py

```diff
@@ -607,10 +607,18 @@ def _compute_cv_scores(self, pipeline, X, y):
         start = time.time()
         cv_data = []
         logger.info("\tStarting cross validation")
+        warnings.filterwarnings("ignore", lineno=665)
```

Comment:
I'm skeptical of `lineno` because it will be impossible to maintain as the size of this file changes. Maybe we don't worry about suppressing warnings?

FWIW, I'm still seeing the warning when I run this code anyways.

![image](https://user-images.githubusercontent.com/41651716/94473187-dd56a800-0199-11eb-808d-8179d6fc0096.png)